ui/src/main/java/com/theoplayer/android/ui/UIController.kt

```diff
@@ -216,27 +204,29 @@ fun UIController(
         )
     )
 
-    PlayerContainer(modifier = modifier, player = player) {
+    PlayerContainer(
+        player = player,
+        modifier = Modifier
+            .background(Color.Black)
+            .then(modifier)
+            .playerAspectRatio(player)
+            .toggleControlsOnTap(
```

Comment:
This is the important bit. The `toggleControlsOnTap` is now on the `PlayerContainer` rather than the `AnimatedContent`, so it will only handle taps *after* all its children (including all of `THEOplayerView`'s own children) have had a chance to handle them.